src/Form/SettingsForm.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\os2forms_dig_sig_server\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Signing settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+  use StringTranslationTrait;
+
+  /**
+   * Name of the config.
+   *
+   * @var string
+   */
+  public static $configName = 'os2forms_dig_sig_server.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'os2forms_dig_sig_server_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [self::$configName];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['signing_service_url'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Signing server URL'),
+      '#default_value' => $this->config(self::$configName)->get('signing_service_url'),
```

Comment:
Consider getting the existing config (i.e. default values) only once, e.g.

```suggestion
    $config = $this->config(self::$configName);
    $form['signing_service_url'] = [
      '#type' => 'textfield',
      '#title' => $this->t('Signing server URL'),
      '#default_value' => $config->get('signing_service_url'),
```